### ISSUE TITLE
Make the Fenix homepage be the contents desktop Bookmarks Toolbar folder.  Reintroduce Tab Groups on desktop as what is now Fenix Collections

### ISSUE THREAD

**Issue by jawz101** [mozilla-mobile/fenix#5853]
### Why/User Benefit/User Problem
Top Sites, Pinned Sites, and Collections are an over-engineering of bookmarks.  The Mobile Bookmarks folder is like an appendage of the desktop bookmarks.  Why not have what is the Bookmarks Toolbar folder on desktop be the main page of the mobile browser?  It would encourage creating Firefox Sync accounts, use of the Bookmarks Toolbar folder on desktop Firefox, and new users would immediately see the cohesive experience between desktop and mobile.  [others with the same sentiment](https://www.reddit.com/r/firefox/comments/dez7hh/does_anyone_actually_use_mobile_bookmarks/)

### What/Requirements
Make the Fenix homepage be the contents of the desktop's Bookmark Toolbar folder.
** as a bonus, you could also reintroduce Tab Groups on the desktop and link them with what is Collections on Fenix.

TLDR- pinned sites sync to Bookmarks Toolbar.  Collections sync as Tab Groups

### Acceptance Criteria (how do I know when I’m done?)
The Fenix homepage is the contents of what is the desktop Firefox's Bookmark Toolbar folder.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-13805)

**Comment by ondondil** [comment]
In my opinion any way of accessing bookmarks from home screen would be welcome because as of now accessing bookmarks takes way to many steps. It could be a new category next to Pinned Sites and Collections. Anything would be better than what we currently have.

**Comment by Dunexus** [comment]
Could the UX team clarify the distinction between collections and bookmarks ?
They seems to achieve the same goal, save and organize links.

**Comment by jawz101** [comment]
In my mind this just flips and scrunches to this: (pardon the crude ms paint.)
![desktop](https://user-images.githubusercontent.com/14151703/66525221-4dc96c80-eae4-11e9-935d-798a71b45aa7.png)

**Comment by madb1lly** [comment]
I use Firefox on Desktop daily at work and at home with a synced account. I never use bookmarks; I use history and the awesome bar to get to where I want to go.
On mobile in Fennec I use the new tab page a lot - pinned sites, top sites, history (especially closed tabs) and synced tabs.

My point is that the suggestion only works if people use desktop bookmarks. In my opinion top sites and pinned sites are a much more flexible way for people to choose what they want to see on their new tab page.

Cheers :)

**Comment by jawz101** [comment]
>I use Firefox on Desktop daily at work and at home with a synced account. I never use bookmarks; I use history and the awesome bar to get to where I want to go.

So your use case would be irrelevant since you don't use the Bookmarks Toolbar on desktop.  But if you did, you could have the same set of icons on both ui's.  They could even make the mobile app icons look big just like the current pinned sites yet it would be a uniform location between mobile and desktop if you have sync enabled.

**Comment by madb1lly** [comment]
Hi @jawz101,

No I don't think my case is irrelevant because what you are asking for will affect everyone. I actually want different sites as my "top" ones on mobile and desktop; some are the same, some are different. 

Personally I prefer Fennec's newtab to what you are proposing, but your proposal would be tolerable (but frustrating).

Cheers :)

**Comment by jawz101** [comment]
@madb1lly Thanks. but from what you said you do not use bookmarks on the desktop.  So if what was the contents of the desktop Bookmarks Toolbar was what shows as Top Sites on mobile, it that would not change anything for you.  That is, the top sites you have set on desktop would remain what they are now.

**Comment by madb1lly** [comment]
Hi @jawz101, maybe I'm missing something from your description. If I don't use the bookmarks toolbar on desktop then what would I see as my main page in Fenix?

**Comment by jawz101** [comment]
> If I don't use the bookmarks toolbar on desktop then what would I see as my main page in Fenix?

If everything else stays as is, the only thing that would change would simply be that pinned sites on mobile would be what is the contents of the desktop's Bookmark Toolbar

**Comment by madb1lly** [comment]
Hi @jawz101,
Okay thanks for the clarification.
If someone doesn't want this, how easy do you think it will be to make it optional? After all, it's part of the sync service, and the number of options for that is already quite complicated.
Cheers :)

**Comment by jawz101** [comment]
This is me just conceptualizing the suggestion.  I don't even know if the developers are reading this suggestion or even thing it's an interesting idea in the 1st place.

**Comment by madb1lly** [comment]
It's good to discuss. I'm sure they're reading everything on GitHub, eventually ;)

Currently on Fennec it's all customisable IIRC, so if they follow that approach you'll be able to restrict it to only what you want.

**Comment by klint** [comment]
My 2cts :) 
Although I agree that Collections should be synced, as mentioned just above, I have very different usages between desktop and mobile browsers, mainly due to screen size and use of mouse vs tactile.
With the following consequences:
- most accessed sites on desktop differ from most accessed sites on mobile
- tab groups on desktop have a different content than collections on mobile (by the way, tab groups are already there on desktop, with Simple Tab Groups powerful addon)
- while hundreds of bookmarks are easily manageable on a desktop screen, being at hand within 2 clicks in the bookmark toolbar for instance,  the number of bookmarks that can be made visible and instantaneously reachable on mobile is far lower due to screen size and information density. Therefore I would not map bookmark toolbar on desktop with pinned sites on mobile.

All that being said, I would more than happy to be able to reach any of the above sets of sites from any synced browser on any device, made available and organized somewhere is the synced bookmarks hierarchy. 
Still, the part of this information that we could see immediately at hand would be different between mobile and desktop.

**Comment by jawz101** [comment]
![Untitled](https://user-images.githubusercontent.com/14151703/80818245-55f11280-8bc2-11ea-98f3-9920f4036cb8.png)

I just use bookmark folders and subfolders within the Bookmarks Toolbar folder on desktop.  And if I right click on a folder and I can open all of the contents within the folder as new tabs.  Bookmark toolbar and creating folders has been the longest-lasting feature amongst all browsers so I don't see why they cannot just take on the same purpose on mobile just in a different presentation.

That's just my thinking but I understand why people appreciate collections and tab groups because I essentially do the same thing- it's just that the mechanics of it all is already there with the bookmark mechanism.

**Comment by JNavas2** [comment]
I too assumed/hoped Collections were the equivalent of Bookmark Groups (folders), greatly enhancing the usefulness of the Fenix home screen. (The few Top Sites is too limited to fit my use case.) But no, **Collections are _saved_ tabs**, complete with history, so when you open them you get _old_ content, not current content–see [Comment](https://github.com/mozilla-mobile/fenix/issues/11506#issuecomment-643264112) to #11254 
> This is intentional behavior with collections currently. We are restoring the state of the tab with history as well.

I don't have a use case for saved tab Collections. (To save and sync tabs I use [archive.today](https://archive.today) and [Google Keep](https://play.google.com/store/apps/details?id=com.google.android.keep&hl=en_US), which syncs to all my devices.) What I want/need is **Bookmark Groups (folders) on the Fenix home screen** including sync. Accessing current Bookmark folders in Fenix is too painful.

**Comment by boek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1813918

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
